ngraph_bridge/ngraph_tensor_manager.cc

```diff
@@ -140,6 +150,47 @@ void NGraphTensorManager::Initialize() {
       FindComplement(m_pipelined_input_indexes, m_prefetched_input_indexes);
 }
 
+//---------------------------------------------------------------------------
+// PrintVector Utility
+//---------------------------------------------------------------------------
+void PrintVector(const vector<int>& input_vector, const string title) {
+  cout << title << endl;
+  std::stringstream ss;
+  for (int val : input_vector) {
+    ss << val << " ";
+  }
+  cout << ss.str() << endl;
+}
```

Comment:
there is ng::join. Maybe we don't need this.

`cout << title << " " << ng::join(input_vector) << "\n";`